flopy/utils/get_modflow.py

```diff
@@ -556,6 +561,10 @@ def add_item(key, fname, do_chmod):
                     items.append(fname)
                     if not fname.endswith(lib_suffix):
                         chmod.add(fname)
+
+            if repo == "modflow6":
```

Comment:
I wasn't able to get this working for the nightly builds, because there doesn't seem to be a reference from the nightly to which git commit of modflow was used to build it.